src/goto-programs/goto_inline_class.h

```diff
@@ -69,6 +69,15 @@ class goto_inlinet
     const inline_mapt &inline_map,
     const bool force_full=false);
 
+  /// Inline specified calls in a goto_program.
+  /// force_full:
+  /// - true:  put skip on recursion and issue warning
+  /// - false: leave call on recursion
+  void goto_inline(
```

Comment:
⛏️ Please turn this into proper doxygen.

Reply:
fixed